matlab/src/Util.h

```diff
@@ -3,34 +3,21 @@
 //
 
 #include "Ice/Ice.h"
-
-#if defined(__GNUC__)
-#    pragma GCC diagnostic ignored "-Wredundant-decls"
-#endif
 #include <mex.h>
 
-typedef struct mxArray_tag mxArray; // Forward declaration to avoid importing mex.h here
-
 namespace IceMatlab
 {
     mxArray* createStringFromUTF8(const std::string&);
     std::string getStringFromUTF16(mxArray*);
-    mxArray* createEmpty();
```

Comment:
All these functions weren't called anywhere, and I am not in favor of keeping unused code just in case we may need it in the future.